run_nodes.sh

```diff
@@ -1,45 +1,84 @@
 #!/bin/bash
 
-if [ -z "$1" ] || (("$1" < 2 || "$1" > 8))
-then
-    echo "The committee size is missing, usage:
+function usage(){
+  echo "Usage:
+      ./run_nodes.sh [-v N_VALIDATORS] [-n N_NON_VALIDATORS] [-b false] [-p BASE_PATH] [ALEPH_NODE_ARG]...
+  where 2 <= N_VALIDATORS <= N_VALIDATORS + N_NON_VALIDATORS <= 10
+  (by default, N_VALIDATORS=4, N_NON_VALIDATORS=0 and BASE_PATH=/tmp)"
+}
 
-    ./run_nodes.sh SIZE [Additional Arguments to ./target/debug/aleph-node]
+N_VALIDATORS=4
+N_NON_VALIDATORS=0
+BUILD_ALEPH_NODE='true'
+BASE_PATH='/tmp'
 
-where 2 <= SIZE <= 8"
-    exit
-fi
+while getopts "v:n:b:p" flag
```

Comment:
missing colon after `p`

```suggestion
while getopts "v:n:b:p:" flag
```